### PR TITLE
Text utilities list

### PR DESCRIPTION
**Pull request by rdthetha**
Added typography and lists

### CODE REVIEW

File: component.css [hunk 1]
```diff
@@ -43,6 +45,7 @@
 
 body {
     background-color: var(--primary-background);
+    font-family: var(--primary-font);
 }
```
Comment (kgajriaAgarwal) [inline component.css:49]: you are mentioning color in font family

**Comment by AshutoshM29** [comment]
Amazing work!! You can merge it

**Comment by yaswanthmyneni** [comment]
code looks good to me.

**Comment by prankurpandeyy** [comment]
the code looks good to me ,there is much space between heading and box contents which is taking width .

**Pull request merged**